roadmap/implementers-guide/src/types/candidate.md

```diff
@@ -172,7 +172,7 @@ struct TransientValidationData {
 	config_max_upward_message_num_per_candidate: u32,
 	/// The number of messages pending of the downward message queue.
 	dmq_length: u32,
-	/// A part of transient validation data related to HRMP.
+	/// A part of transient validaiton data related to HRMP.
```

Comment:
```suggestion
	/// A part of transient validation data related to HRMP.
```

Give https://github.com/drahnr/cargo-spellcheck a shot, maybe it works for you as well as it does for me :)

Reply:
ouch that's embarrassing

thanks for the pointer, will try it!